Sources/MapboxNavigation/Constants.swift

```diff
@@ -40,6 +40,11 @@ public let CongestionAttribute = "congestion"
  */
 public let NavigationViewMinimumVolumeForWarning: Float = 0.3
 
+/**
+ The distance of blending color between two different congestion level segment in meters.
+ */
+public var GradientCongestionblendingDistance: Double = 30.0
```

Comment:
Think you missed a capital here:
```suggestion
public var GradientCongestionBlendingDistance: Double = 30.0
```

Reply:
Thanks for the suggestion, I'll update this name.